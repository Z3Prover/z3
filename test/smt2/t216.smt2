
; Copyright (c) 2015 Microsoft Corporation

(set-option :pp.decimal true)
(set-option :pp.decimal-precision 20)

(simplify (+ (^ 3.0 (/ 1.0 2.0)) (^ 5.0 (/ 1.0 2.0))))
(simplify (^ (* 2.0 (+ 4.0 (^ 15.0 (/ 1.0 2.0)))) (/ 1.0 2.0)))
(simplify (= (+ (^ 3.0 (/ 1.0 2.0)) (^ 5.0 (/ 1.0 2.0)))
             (^ (* 2.0 (+ 4.0 (^ 15.0 (/ 1.0 2.0)))) (/ 1.0 2.0))))

(simplify (- (^ 3.0 (/ 1.0 2.0)) (^ 5.0 (/ 1.0 2.0))))
(simplify (- (^ (* 2.0 (- 4.0 (^ 15.0 (/ 1.0 2.0)))) (/ 1.0 2.0))))
(simplify (= (- (^ 3.0 (/ 1.0 2.0)) (^ 5.0 (/ 1.0 2.0)))
             (- (^ (* 2.0 (- 4.0 (^ 15.0 (/ 1.0 2.0)))) (/ 1.0 2.0)))))

(simplify (- (^ 3.0 (/ 1.0 2.0)) (* 2.0 (^ 2.0 (/ 1.0 2.0)))))
(simplify (- (^ (- 11.0 (* 4.0 (^ 6.0 (/ 1.0 2.0)))) (/ 1.0 2.0))))
(simplify (= (- (^ 3.0 (/ 1.0 2.0)) (* 2.0 (^ 2.0 (/ 1.0 2.0))))
             (- (^ (- 11.0 (* 4.0 (^ 6.0 (/ 1.0 2.0)))) (/ 1.0 2.0)))))

(simplify (^ 5.0 (/ 125.0 100.0)))