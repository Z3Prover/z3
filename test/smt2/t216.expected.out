3.96811878506866698993?
3.96811878506866698993?
true
(- 0.50401716993091240288?)
(- 0.50401716993091240288?)
true
(- 1.09637631717731280407?)
(- 1.09637631717731280407?)
true
7.47674390610610270955?
